source/index.html

```diff
@@ -1,12 +1,118 @@
+              <h3 class="features__title">Настоящий городок</h3>
+              <p features__number>— №1 —</p>
+              <p features__description>Седона не аттракцион для туристов, там течет своя жизнь</p>
+            </div>
+            <div class="features__image features__image--city"></div>
+            <ul class="features__recommendation-list">
+              <li class="recommendation-list__item recommendation-list__item--housing">
+                <h4 class="recommendation-list__header">Жилье</h4>
+                <p class="recommendation-list__description">Рекомендуем пожить в настоящем мотеле, все как в кино!</p>
+              </li>
+              <li class="recommendation-list__item recommendation-list__item--food">
+                <h4 class="recommendation-list__header">Еда</h4>
+                <p class="recommendation-list__description">Всегда заказывайте фирменный бургер, вы не разочаруетесь!
+                </p>
+              </li>
+              <li class="recommendation-list__item recommendation-list__item--souvenirs">
+                <h4 class="recommendation-list__header">Сувениры</h4>
+                <p class="recommendation-list__description">Не только китайского, но и местного производства!</p>
+              </li>
+            </ul>
+          </li>
+          <li class="features__item">
+            <div class="features__card features__card--main">
+              <h3 class="features__title">Там есть мост дьявола</h3>
+              <p features__number>— №2 —</p>
+              <p features__description>Да, по нему можно пройти! Если вы осмелитесь, конечно</p>
+            </div>
+            <div class="features__image features__image--bridge"></div>
+          </li>
+          <li class="features__item features__card features__card--sub">
+            <h3 class="features__title">Небольшая площадь</h3>
+            <p features__number>— №3 —</p>
+            <p features__description>Все интересные места находятся очень близко</p>
+          </li>
+          <li class="features__item features__card features__card--sub">
+            <h3 class="features__title">Красивая дорога</h3>
+            <p features__number>— №4 —</p>
+            <p features__description>Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+          </li>
+          <li class="features__item features__card features__card--sub">
+            <h3 class="features__title">Мало туристов</h3>
+            <p features__number>— №5 —</p>
+            <p features__description>Большинство едет в гранд каньон и толпится там</p>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="booking">
+      <h2 class="visually-hidden">Поиск гостиницы</h2>
+      <p class="booking__title">Заинтересовались?</p>
+      <p class="booking__description">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц
+        в Cедоне</p>
+      <a href="#" class="button booking__button">Найти гостиницу</a>
```

Comment:
Читаемости ради стоит сперва указывать БЭМ-элемент, а потом миксовый БЭМ-блок
```suggestion
      <a href="#" class="booking__button button">Найти гостиницу</a>
```
Также не стоит смешивать кнопки с ссылками. тег `a` именованный как `button` может ввести разработчика в заблуждение так же как тег `button` именованный как `link`

Reply:
Если кнопки и некоторые ссылки имеют одинаковую стилизацию, то можно миксовать классы ссылки как class="button block__link"?

Reply:
Можно